modules/weko-workflow/weko_workflow/utils.py

```diff
@@ -1525,3 +1528,93 @@ def update_indexes_public_state(item_id):
                 index.public_state = True
     if update_db:
         db.session.commit()
+
+
+def get_account_info(user_id):
+    """Get account's info: email, username.
+
+    :param user_id: User id.
+
+    :return: email, username.
+    """
+    data = get_user_profile_info(user_id)
+    if data:
+        return data.get('subitem_mail_address'), \
+            data.get('subitem_displayname')
+    else:
+        return None, None
+
+
+def get_name_display_hide(list_hide, role):
```

Comment:
This function don't have re-usability, so you can put it a @staticmethod of FlowSettingView class.

Thanks,

Reply:
Has been fixed.